tensorboard/webapp/core/store/core_selectors.ts

```diff
@@ -82,9 +82,23 @@ export const getRuns = createSelector(
   }
 );
 
-export const getRunSelection = createSelector(
+const selectSelection = createSelector(
   selectCoreState,
-  (state: CoreState): Map<RunId, boolean> => {
+  (state: CoreState): Set<RunId> => {
     return state.polymerInteropRunSelection;
   }
 );
+
+export const getRunSelection = createSelector(
+  getRuns,
+  selectSelection,
+  (runs: Run[], selection: Set<RunId>): Map<RunId, boolean> => {
```

Comment:
The internal run store's version of this selector returns `Record<string, boolean>|null`, which is almost the same except for the `|null`.

Should we make this OSS return type nullable, so that we don't have to add guards at all the OSS locations calling this selector later on?